debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -735,6 +735,25 @@ public JdbcConnection prepareQuery(String preparedQueryString, StatementPreparer
         return this;
     }
 
+    /**
+     * Execute a SQL prepared query.
+     *
+     * @param preparedQueryString the prepared query string
+     * @param preparer the function that supplied arguments to the prepared statement; may not be null
+     * @return this object for chaining methods together
+     * @throws SQLException if there is an error connecting to the database or executing the statements
+     * @see #execute(Operations)
+     */
+    public JdbcConnection prepareQuery(String preparedQueryString, StatementPreparer preparer)
```

Comment:
It looks like `prepareUpdate()` already does the same. Although the name doesn't seem suitable to the use case at hand, it's consistent with the method naming in JDBC.

Reply:
I'll switch it over. I noticed it as well but added the other method due to the name.